File: app/src/main/java/awais/instagrabber/repositories/responses/directmessages/DirectItemVoiceMedia.java
package awais.instagrabber.repositories.responses.directmessages;

import awais.instagrabber.repositories.responses.Media;

public class DirectItemVoiceMedia {
    private final Media media;
    private final int seenCount;
    private final String viewMode;

    public DirectItemVoiceMedia(final Media media, final int seenCount, final String viewMode) {
        this.media = media;
        this.seenCount = seenCount;
        this.viewMode = viewMode;
    }

    public Media getMedia() {
        return media;
    }

    public int getSeenCount() {
        return seenCount;
    }

    public String getViewMode() {
        return viewMode;
    }
}

File: app/src/main/java/awais/instagrabber/repositories/responses/directmessages/DirectItemVoiceMedia.kt
package awais.instagrabber.repositories.responses.directmessages

import awais.instagrabber.repositories.responses.Media
import java.io.Serializable

data class DirectItemVoiceMedia(
    val media: Media? = null,
    val seenCount: Int = 0,
    val viewMode: String? = null,
) : Serializable